### PR TITLE
Load navigation chunks on console.dev

### PR DESCRIPTION
**Pull request by karelhala**
### Description

console.dev.redhat.com recently stopped working because navigation chunks were not loaded. This PR disables unleash check for console.dev origin.

### CODE REVIEW

File: src/utils/useNavigation.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ const useNavigation = () => {
     let observer: MutationObserver | undefined;
     // reset no nav flag
     setNoNav(false);
-    if (currentNamespace && (flagsReady || flagsError)) {
+    if (currentNamespace && (flagsReady || flagsError || location.origin.includes('console.dev.rehdat.com'))) {
```
Comment (flacatus) [inline src/utils/useNavigation.ts:106]: console.dev.rehdat.com => console.dev.redhat.com ?

**Comment by Hyperkid123** [comment]
@karelhala the unleash check should be disabled dif it fails to retrieve the data and acts as if all flags were false. I think you might have encountered a scenario that the current check did not cover.

**Comment by karelhala** [comment]
@Hyperkid123 the problem here is that the unleash is not deployed on console.dev.redhat.com and fails to load the navigation. I was trying to make this in a hotfix manner, but I'll probably launch chrome locally, change the url for unleash and do a proper fix.

**Comment by Hyperkid123** [comment]
The fetch handler should be able to resolve 404s with no issues. Maybe we are getting different error?

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**